plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
still don't get it: As the timeout is part of a DC, I suppose it applies only to this very DC, not to the sum of all DCs. So the number of DCs deployed doesn't matter.

The same argument goes for 'you can overwrite if required' can be applied in the other direction as well (when you need a higher timeout, why not simply configure it ?). It's simply about which use case is more likely (with such a high timeout everybody doesn't get a feedback early when a deployment fails for another reason than long image download times, and as said, wonder why we should be cleverer than the OpenSift guys and don't rely on the OpenShift timeout ?)

It's really not about nit-picking, but I really want to understand the issue better (i.e. why the timeout should be applied to all DCs together and not individually). 

Having 1 hour for downloading image feels a bit much for me, even for slow wifi. About what number are we talking ? E.g. for an 2 MBit WiFi and a 500 MB image, this would take 33 mins (if everything needs to be fetched from scratch). 

Within 1 hour you can download a 900 MB image (2 MBit) or 500 MByte with a 1.1 MBit speed. Is this it the usual user then I'm fine with the timeout. But even then I still doubt that the user is willing to wait 1 hour without any progress indicator. More likely he will kill the process before (and since the download continues in the background, the next time it's more likely to succeed faster.

Said that I will add the 1 hour timeout now. Let's wait on user's feedback, as this is the only way to find out the real thing.